api/db/changelog/db.changelog-master.xml

```diff
@@ -168,6 +168,7 @@
   <include file="changelog/db.changelog-156-billing-buffer-entry-status-index.xml"/>
   <include file="changelog/db.changelog-157-add-user-access-tier.xml"/>
   <include file="changelog/db.changelog-158-workspace-add-google-project.xml"/>
+  <include file="changelog/db.changelog-160-delete-data-dictionary-entry-table.xml"/>
```

Comment:
Should this be 159?

Reply:
You are right

Reply:
So the new implementation (CDR) is already in PROD as of the last release. So i am going ahead with this merge